test/models/user_test.rb

```diff
@@ -62,11 +62,21 @@ class UserTest < ActiveSupport::TestCase
     assert_equal 2, user.total_learning_time
   end
 
-  test "dates_from_start_lerning" do
+  test "dates_from_start_learning" do
     user = users(:komagata)
     user.created_at = Time.new(2019, 1, 1, 0, 0, 0)
     travel_to Time.new(2020, 1, 1, 0, 0, 0) do
-      assert_equal 365, user.dates_from_start_lerning
+      assert_equal 365, user.dates_from_start_learning
     end
   end
+
+  test "order_by_counts" do
+    @users = User.order_by_counts("report", "desc")
```

Comment:
ivarにする必要がありますか？

```suggestion
    names = User.order_by_counts("report", "desc").pluck(:login_name)
```

で良さそうです